core/spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/DefaultBinderFactory.java

```diff
@@ -302,99 +304,61 @@ private <T> Binder<T, ConsumerProperties, ProducerProperties> getBinderInstance(
 			this.binderInstanceCache.put(configurationName,
 					new SimpleImmutableEntry<>(binder, binderProducingContext));
 		}
-		logger.info("Retrieving cached binder: " + configurationName);
+		logger.trace("Retrieving cached binder: " + configurationName);
```

Comment:
And this one better to use with a lambda 😄

Reply:
I resisted the urge of switching the DBF to use the new log accessor so that is not an option in this PR.